src/sc_crawler/tables.py

```
@@ -153,9 +136,30 @@ def _get_methods(self):
+                    "inventory_zones",
+                    "inventory_servers",
+                    "inventory_server_prices",
+                    "inventory_server_prices_spot",
+                    "inventory_storage_prices",
+                    "inventory_traffic_prices",
+                    "inventory_ipv4_prices",
+                ]:
+                    if method not in methods:
+                        raise NotImplementedError(
+                            f"Unsupported '{self.vendor_id}' vendor: missing '{method}' method."
+                        )
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)